charts/kube-master/templates/controller-manager.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: "extensions/v1beta1"
 kind: Deployment
 metadata:
-  name: {{ include "master.fullname" . }}-c-manager
+  name: {{ include "master.fullname" . }}-cmanager
```

Comment:
By renaming the deployment we should handle the upgrade case better for exiting klusters. The old deployment will be deleted and a new one will be created. This should prevent the we leave orphaned replicates/pods behind as explained here https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#label-selector-updates